ceph-setup/build/build

```diff
@@ -53,6 +53,25 @@ else
     export CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=tcmalloc"
 fi
 
+
+# When using autotools/autoconf it is possible to see output from `git diff`
```

Comment:
if we're gonna add three lines of comment (great!) let's actually call specific attention to "autogen.sh calls aclocal with --install, which can..." for documentation/archaeologists' sakes